api/installation_service.go

```diff
@@ -52,3 +60,38 @@ func (is InstallationService) Export(outputFile string) error {
 
 	return nil
 }
+
+func (is InstallationService) Import(input ImportInstallationInput) error {
```

Comment:
We seem to be missing logic that checks as to whether the Ops Manager is already configured. If it is, this should error in a more graceful way than `422`